frontend/src/scenes/insights/insightCommandLogic.ts

```diff
@@ -3,15 +3,24 @@ import { kea } from 'kea'
 import { insightCommandLogicType } from './insightCommandLogicType'
 import { compareFilterLogic } from 'lib/components/CompareFilter/compareFilterLogic'
 import { RiseOutlined } from '@ant-design/icons'
-import { insightDateFilterLogic } from 'scenes/insights/InsightDateFilter/insightDateFilterLogic'
 import { dateMapping } from 'lib/utils'
+import { InsightLogicProps } from '~/types'
+import { keyForInsightLogicProps } from 'scenes/insights/sharedUtils'
+import { insightDateFilterLogic } from 'scenes/insights/InsightDateFilter/insightDateFilterLogic'
 
 const INSIGHT_COMMAND_SCOPE = 'insights'
 
 export const insightCommandLogic = kea<insightCommandLogicType>({
```

Comment:
Does it make sense to mount a keyed insightCommandLogic for each insight instance? Is there a chance multiple insightCommandLogic instances will be mounted simultaneously and cause weird behavior?

Reply:
Indeed, this could cause problems. I added:

```ts
if (!props.syncWithUrl) {
    return
}
```
to the beginning of this logic, so it won't register any commands on boot.